lib/bolt/bolt_option_parser.rb

```diff
@@ -803,8 +803,15 @@ def initialize(options)
       end
       define('--debug', 'Display debug logging') do |_|
         @options[:debug] = true
+        # We don't actually set '--log-level debug' here, but once the options are evaluated by
+        # the config class the end result is the same.
+        @warnings << { msg: "Command line option '--debug' is deprecated, setting '--log-level debug' instead." }
+      end
+      define('--log-level LEVEL',
+             "Set the log level for the console. Available options are",
+             "debug, info, notice, warn, error, fatal, any.") do |level|
+        @options[:log] = { 'console' => { 'level' => level } }
```

Comment:
Is this before config is loaded? Want to make sure this doesn't override file logging from a config file.

Reply:
It happens before config is loaded, but all of our CLI options override logging from a config file. `log` specifically is merged, so it will only affect the config for `console`.